### ISSUE TITLE
More comprehensive modes for notifications

### ISSUE THREAD

**Issue by Amolith** [ubports/ubuntu-touch#697]
I'd like a better way to control notifications. A couple of different modes for this would be nice. For example:

* Sound off/vibrate on
* Sound off/vibrate on
* Sound off/vibrate off
  * Except for alarms
  * Complete silence

**Comment by Amolith** [comment]
> That's two times the same
@doniks I fixed it. Thank you for pointing that out!

**Comment by mateosalta** [comment]
Yes, I have notifications I would like to see in the notification list, but not have them alert me -  or flash the light.

**Comment by userj** [comment]
I am in favor of any "feature" that helps to protect/improve the human. 
Unwanted sound, vibration, light seem to be important things to consider in that area.
